mage.go

```diff
@@ -0,0 +1,49 @@
+//go:build mage
+
+package main
+
+import (
+	"fmt"
+	"github.com/magefile/mage/sh"
+)
+
+const versionStringEnvVarName = "POCKET_VERSION_STRING"
+const pocketPackage = "github.com/pokt-network/pocket/cmd/pocket"
+const ldflag = "-X main.version=$" + versionStringEnvVarName
```

Comment:
s/ldflag/slFlag

Reply:
explicitly ldflag, because it is the "value" part of the `-ldflags` argument to `go build`. Perhaps it should be `ldflags`, though.

Reply:
+1 to `ldflags`